src/lib.rs

```diff
@@ -1839,6 +1842,43 @@ impl OpAttrDef {
     }
 }
 
+/// Types of attribute values for an operation
+#[non_exhaustive]
+#[derive(Debug)]
+pub enum AttrValue {
+    S(Vec<u8>), 
+    I(i64), 
+    F(f32), 
+    B(bool), 
+    Field_type(DataType), 
+    Shape(Shape), 
+    Tensor(protos::tensor::TensorProto), 
```

Comment:
The type of the `Tensor` variant is also something we can change. There is a `Tensor` type in `src/lib.rs` but it requires us to mention a datatype when defining it.

https://github.com/tensorflow/rust/blob/ceefb76ad8aa7b7e2fe0724279fead86ffe35539/src/lib.rs#L1257-L1260

Reply:
I meant to mention `AnyTensor`, but it looks like I forgot.  We will probably need to make it public and then this can be `Box<dyn AnyTensor>`.